documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -679,8 +679,8 @@ First, let Hibernate know that you want to use the OGM Infinispan Remote datasto
 The next step is to configure the Hot Rod client.
 You have two options:
```

Comment:
removing `either / or` maybe we can add an `alternative` to the word options

Reply:
I will replace `options` with `alternatives`. I don't like too much having a list and also using either/or. It's just my personal preference